lib/install/data/data.go

```diff
@@ -190,4 +190,8 @@ func (d *Data) CopyNonEmpty(src *Data) {
 		d.ContainerNetworks = src.ContainerNetworks
 	}
 	d.Timeout = src.Timeout
+	if src.HTTPProxy != nil || src.HTTPSProxy != nil {
+		d.HTTPProxy = src.HTTPProxy
```

Comment:
Just for my understanding: is it ok to update both `d.HTTPProxy` and `d.HTTPSProxy` if either of them are non-nil?